src/transformers/modeling_ctrl.py

```diff
@@ -30,7 +30,9 @@
 
 logger = logging.getLogger(__name__)
 
-CTRL_PRETRAINED_MODEL_ARCHIVE_MAP = {"ctrl": "https://storage.googleapis.com/sf-ctrl/pytorch/seqlen256_v1.bin"}
+CTRL_PRETRAINED_MODEL_ARCHIVE_MAP = {
+    "ctrl": "https://s3.amazonaws.com/models.huggingface.co/bert/ctrl-pytorch_model.bin"
```

Comment:
anyway we could upload to the salesforce/ctrl namespace so that we'll get a nice webpage under the salesforce org?